src/sandstorm/appid-replacements.capnp

```diff
@@ -0,0 +1,84 @@
+# Sandstorm - Personal Cloud Sandbox
+# Copyright (c) 2016 Sandstorm Development Group, Inc. and contributors
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+@0xa53cae3f717a1676;
+# This file is used to recover from lost or stolen app keys.
+#
+# If you are an app developer and you have lost access to your key or suspect it may have been
```

Comment:
This probably needs to also make it into the docs, and be adapted for `vagrant-spk`, even if that's just saying `vagrant-spk ssh` first and specifying `--keyring /host-dot-sandstorm/sandstorm-keyring`

cc @paulproteus